lib/promises.helper.js

```diff
@@ -43,13 +44,24 @@ function handlePromise(parameterIndex, original) {
     return fn;
 }
 
-prototype = jasmine.Env.prototype;
+jasmineEnv = jasmine.currentEnv_;
```

Comment:
From what I read, this is the current Env instance, which can get thrown away.  We want the Env prototype/class directly.